docs/docs/infrahubctl.mdx

```diff
@@ -32,10 +32,12 @@ The `infrahubctl` command line utility is installed as a part of the [Infrahub S
 | `INFRAHUB_API_TOKEN`      | `06438eb2-8019-4776-878c-0941b1f1d1ec` |
 | `INFRAHUB_DEFAULT_BRANCH` | main                                   |
 
-> You can also provide the location of a configuration file via the environment variable `INFRAHUBCTL_CONFIG`.
+You can obtain the `INFRAHUB_API_TOKEN` from the web GUI by navigating to 'Account Settings' under your profile and selecting 'Tokens'. You can also manage the tokens via GraphQL queries. You can find more information on how to do that [here](/guides/managing-api-tokens).
```

Comment:
I'd just say "don't have a token yet? you can find more information on how to manage API token in this guide". So if one day we change that part of the product we can change only the directly related documentation

Reply:
Good point, I will change it.